src/universal/modules/meeting/containers/MeetingAgendaItems/MeetingAgendaItemsContainer.js

```diff
@@ -30,7 +30,10 @@ const mapStateToProps = (state, props) => {
   const {agenda} = cashay.query(meetingAgendaItemsQuery, {
     op: 'meetingAgendaItemsContainer',
     key: teamId,
-    variables: {teamId}
+    variables: {teamId},
+    sort: {
```

Comment:
Ha! Always the simple stuff. Pretty nice that sort operator is there on Cashay, huh?